guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -625,10 +625,10 @@ WEBVTT
 [J Jay Couey]: And so it is very important to understand that that premise
 
 24:01.263 --> 24:12.075
-[J Jay Couey]: That premise is central to Biology 101 at every university in the Western world, and it is absolutely central to the idea that the Human Genome Project accomplished anything at all.
+[J Jay Couey]: that premise is central to Biology 101 at every university in the Western world, and it is absolutely central to the idea that the Human Genome Project accomplished anything at all.
```

Comment:
I had to double check the video for this one. Seemed odd that there was that many "that"s... but he was finding a book.